source/ShipInfoDisplay.cpp

```diff
@@ -615,29 +615,33 @@ void ShipInfoDisplay::DrawShipHealthStats(const Ship &ship, const Rectangle & bo
 
 
 	// This allows the section to stack nicely with other info panel sections,
-	// But will also allow it to be called on its own in a new box if desire.
+	// But will also allow it to be called on its own in a new box if desired.
```

Comment:
I mean, while we're changing this line...
```suggestion
	// but will also allow it to be called on its own in a new box if desired.
```